frontend/src/app/core/helpers/app-domain-helper.ts

```diff
@@ -0,0 +1,5 @@
+export const getAppDomain = (): string => {
```

Comment:
Probably you can use document.location.hostname to get appp domain?

Reply:
why do we need this helper?